### DISCUSSION TITLE
키퍼 홈페이지 개발 회고글~

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#148] [5. 회고]
https://velog.io/@shkisme/%EB%AA%A8%EB%93%A0-%EA%B2%83%EC%9D%80-Trade-Off-%EC%83%81%ED%99%A9%EC%97%90-%EB%A7%9E%EA%B2%8C

홈페이지 개발 회고 글을 써봤습니다.

다음 주제로는

- orphanRemoval 로 인한 성능 문제
- 백엔드 개발자로 취직하려면 어떤 프로젝트를 하고, 어떤 포트폴리오를 만드는 게 좋을까? 고찰

에 대한 글을 써보려고 합니다~~

**Comment by 02ggang9** [comment]
글 잘 읽었습니다..! 적어주신대로 빨리 완성하고 성능 개선이랑 다른 기술 이것 저것 해보고 싶네요..!!
정말 많이 배우고 있습니당
